File: sdkwork-backend-sdk-csharp/Models/ChatCompletionChunk.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Backend.Models
{
    public class ChatCompletionChunk
    {
        public string? Id { get; set; }
        public string? Object { get; set; }
        public int? Created { get; set; }
        public string? Model { get; set; }
        public List<ChatCompletionChunkChoice>? Choices { get; set; }
        public CompletionUsage? Usage { get; set; }
        public ChatCompletionMetadata? Metadata { get; set; }
        public string? Content { get; set; }
        public PlusAgent? Agent { get; set; }
        public string? ReasoningContent { get; set; }
        public string? SystemFingerprint { get; set; }
    }
}

File: sdkwork-backend-sdk-go/types/chat_completion_chunk.go
package types

// 聊天完成Stream方式的响应对象，用于流式传输的聊天模型响应
type ChatCompletionChunk struct {
	Id string `json:"id"`
	Object string `json:"object"`
	Created int `json:"created"`
	Model string `json:"model"`
	Choices []ChatCompletionChunkChoice `json:"choices"`
	Usage CompletionUsage `json:"usage"`
	Metadata ChatCompletionMetadata `json:"metadata"`
	Content string `json:"content"`
	Agent PlusAgent `json:"agent"`
	ReasoningContent string `json:"reasoningContent"`
	SystemFingerprint string `json:"system_fingerprint"`
}
